src/DGtal/io/viewers/Viewer3D.ih

```diff
@@ -820,7 +820,7 @@ DGtal::Viewer3D<Space, KSpace>::init()
   myGLScaleFactorY=1.0;
   myGLScaleFactorZ=1.0;
   myGLLineMinWidth=1.5;
-
+  setBackgroundColor ( QColor ( 217, 228, 255 ) );
```

Comment:
The white background is still needed for exports and snapshots. 
The idea of this PR is to replace the dark background (which is not great when displaying isothetic quads with classical lightning) by a "lighter" one.